L1Trigger/TrackerTFP/test/test_cfg.py

```diff
@@ -37,24 +37,44 @@
 process.load( 'L1Trigger.TrackerTFP.Analyzer_cff' )
 
 # build schedule
-process.mc = cms.Sequence( process.StubAssociator )
-process.dtc = cms.Sequence( process.TrackerDTCProducer + process.TrackerDTCAnalyzer )
-process.gp = cms.Sequence( process.TrackerTFPProducerGP + process.TrackerTFPAnalyzerGP )
-process.ht = cms.Sequence( process.TrackerTFPProducerHT + process.TrackerTFPAnalyzerHT )
-process.mht = cms.Sequence( process.TrackerTFPProducerMHT + process.TrackerTFPAnalyzerMHT )
-process.zht = cms.Sequence( process.TrackerTFPProducerZHT + process.TrackerTFPAnalyzerZHT )
-process.interIn = cms.Sequence( process.TrackerTFPProducerZHTout + process.TrackerTFPProducerKFin + process.TrackerTFPAnalyzerKFin )
-process.kf = cms.Sequence( process.TrackerTFPProducerKF + process.TrackerTFPAnalyzerKF )
-process.interOut = cms.Sequence( process.TrackerTFPProducerTT + process.TrackerTFPProducerAS )#+ process.TrackerTFPAnalyzerTT )
-process.tt = cms.Path( process.mc + process.dtc + process.gp + process.ht + process.mht + process.zht + process.interIn + process.kf )#+ process.interOut )
+process.mc  = cms.Sequence( process.StubAssociator )
+process.dtc = cms.Sequence( process.ProducerDTC + process.AnalyzerDTC )
+process.pp  = cms.Sequence( process.ProducerPP                        )
+process.gp  = cms.Sequence( process.ProducerGP  + process.AnalyzerGP  )
+process.ht  = cms.Sequence( process.ProducerHT  + process.AnalyzerHT  )
+process.ctb = cms.Sequence( process.ProducerCTB + process.AnalyzerCTB )
+process.kf  = cms.Sequence( process.ProducerKF  + process.AnalyzerKF  )
+process.dr  = cms.Sequence( process.ProducerDR  + process.AnalyzerDR  )
+process.tq  = cms.Sequence( process.ProducerTQ                        )
+process.tfp = cms.Sequence( process.ProducerTFP                       )
+process.tt  = cms.Path( process.mc + process.dtc + process.pp + process.gp + process.ht + process.ctb + process.kf + process.dr + process.tq )# + process.tfp )
 process.schedule = cms.Schedule( process.tt )
 
 # create options
 import FWCore.ParameterSet.VarParsing as VarParsing
 options = VarParsing.VarParsing( 'analysis' )
 # specify input MC
 Samples = [
-'/store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/0959f326-3f52-48d8-9fcf-65fc41de4e27.root'
+'/store/relval/CMSSW_14_0_0_pre2/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_133X_mcRun4_realistic_v1_STD_2026D98_PU200_RV229-v1/2580000/0b2b0b0b-f312-48a8-9d46-ccbadc69bbfd.root',
```

Comment:
Do we really need dozens of root files here? Consider just one as an example. Same comment applies elsewhere.

Reply:
done